cmd/ctr/commands/snapshots/snapshots.go

```diff
@@ -129,7 +129,7 @@ var diffCommand = cli.Command{
 		}
 
 		if idB == "" {
-			desc, err = rootfs.Diff(ctx, idA, snapshotter, client.DiffService(), opts...)
+			desc, err = rootfs.CreateDiff(ctx, idA, snapshotter, client.DiffService(), opts...)
```

Comment:
This went from a verb, diff, to creating a noun. I think that we generally use diff as a verb. Any reason for this change?

Reply:
Yes, trying to be more consistent. We already use `diff` as a noun to represent a filesystem difference. The verb also being `diff` makes it even more confusing, the artifact of a `diff` is a `diff`... rather the artifact of comparing two filesystems is a `diff`, then the action can either be to compare two file systems or to create a `diff` artifact.